CHANGELOG.md

```diff
@@ -81,6 +81,8 @@ and this project adheres to
   - [#3453](https://github.com/bpftrace/bpftrace/pull/3453)
 - Fix json formatting for hex values
   - [#3475](https://github.com/bpftrace/bpftrace/pull/3475)
+- Fix binary operations on integers always returning 64 bit values
```

Comment:
If this PR was the cause then it hasn't been release yet and this probably doesn't need a changelog entry: https://github.com/bpftrace/bpftrace/pull/3416

Reply:
While the referenced PR does cause the breakage, I think this was already the case. So a Fixes: entry makes sense to me:

```
$ sudo bpftrace -e 'BEGIN { print(sizeof((uint32)5 + (uint32)5)) }'
Attaching 1 probe...
8
^C
```